pyproject.toml

```diff
@@ -1,3 +1,76 @@
+[build-system]
+requires = ["setuptools>=61.2"]
```

Comment:
I recommend setuptools >= 64 since that release added editable installs for PEP 621 based projects.

Reply:
```suggestion
requires = ["setuptools>=64"]
```